lm_eval/tasks/__init__.py

```diff
@@ -214,6 +214,7 @@ def initialize_tasks(verbosity="INFO"):
 
 def get_task(task_name, config):
     try:
+        print(task_name, config)
```

Comment:
Please remove this print before merging!

Reply:
I'm so sorry forgot this print here.